packages/airnode-protocol/contracts/rrp/authorizers/admin/AirnodeRequesterAuthorizerRegistryClient.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.9;
+
+import "./interfaces/IAirnodeRequesterAuthorizerRegistryClient.sol";
+
+contract AirnodeRequesterAuthorizerRegistryClient is
+    IAirnodeRequesterAuthorizerRegistryClient
+{
+    /// @notice Address of the AirnodeRequesterAuthorizerRegistry contract that keeps the
+    /// RequesterAuthorizerWithManager addresses for different chains.
+    address public immutable override airnodeRequesterAuthorizerRegistry;
```

Comment:
Should this really be immutable? I'm wondering if it would ever make sense to set a new  registry contract address in the future via setter function

Reply:
Also, we do have a setter for AirnodeFeeRegistry in AirnodeTokenLock